packages/edit-site/src/components/block-editor/block-editor-provider/default-block-editor-provider.js

```diff
@@ -59,7 +60,7 @@ export default function DefaultBlockEditorProvider( { children } ) {
 
 	return (
 		<ExperimentalBlockEditorProvider
-			settings={ settings }
+			settings={ { ...settings, blockPreview: BlockPreview } }
```

Comment:
We use an alternative BlockPreview that renders using WP REST API for WP specific editors, but if no alternative block previewer is offered, we default to rendering using a rendered block editor.